components/integrations/youtube/YouTubeFeed.tsx

```diff
@@ -44,20 +51,128 @@ const YouTubeFeed = ({
+                >
+                  <feMorphology
+                    radius=".5"
+                    operator="dilate"
+                    in="SourceAlpha"
+                    result="shadowSpreadOuter1"
+                  />
+                  <feOffset in="shadowSpreadOuter1" result="shadowOffsetOuter1" />
+                  <feGaussianBlur
+                    stdDeviation="2.5"
+                    in="shadowOffsetOuter1"
+                    result="shadowBlurOuter1"
+                  />
+                  <feColorMatrix
+                    values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"
+                    in="shadowBlurOuter1"
+                  />
+                </filter>
+                <filter
+                  x="-49.3%"
+                  y="-45.7%"
+                  width="198.6%"
+                  height="198.6%"
+                  filterUnits="objectBoundingBox"
+                  id="c"
+                >
+                  <feMorphology
+                    radius=".5"
+                    operator="dilate"
+                    in="SourceAlpha"
+                    result="shadowSpreadOuter1"
+                  />
+                  <feOffset in="shadowSpreadOuter1" result="shadowOffsetOuter1" />
+                  <feGaussianBlur
+                    stdDeviation="2.5"
+                    in="shadowOffsetOuter1"
+                    result="shadowBlurOuter1"
+                  />
+                  <feColorMatrix
+                    values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"
+                    in="shadowBlurOuter1"
+                  />
+                </filter>
+                <path
+                  d="M24.5 0C38.031 0 49 10.969 49 24.5S38.031 49 24.5 49 0 38.031 0 24.5 10.969 0 24.5 0zm0 4.712c-10.929 0-19.788 8.86-19.788 19.788 0 10.929 8.86 19.788 19.788 19.788 10.929 0 19.788-8.86 19.788-19.788 0-10.929-8.86-19.788-19.788-19.788z"
+                  id="b"
+                />
+                <path
+                  d="m27.394 17.789 6.882 13.764A1 1 0 0 1 33.382 33H19.618a1 1 0 0 1-.894-1.447l6.882-13.764a1 1 0 0 1 1.788 0z"
+                  id="d"
+                />
+              </defs>
+              <g fill="none" fillRule="evenodd">
+                <g fillRule="nonzero" transform="translate(5 5)">
+                  <use fill="#000" filter="url(#a)" xlinkHref="#b" />
+                  <use fill="currentColor" xlinkHref="#b" />
+                </g>
+                <g transform="rotate(90 26.5 29.5)">
+                  <use fill="#000" filter="url(#c)" xlinkHref="#d" />
+                  <use fill="currentColor" xlinkHref="#d" />
+                </g>
+              </g>
+            </svg>
+            <Image src={snippet.thumbnails.medium.url} layout="fill" />
+          </button>
+        ) : (
+          <iframe
+            className={classnames('w-full', 'bg-gray-light')}
+            src={`https://www.youtube.com/embed/${snippet.resourceId.videoId}?enablejsapi=1`}
```

Comment:
If we want this to autoplay without implementing the iframe API, we'll need to add `&autoplay=1&mute=1` to the src.